src/index.ts

```diff
@@ -1,21 +1,93 @@
+
+import {
```

Comment:
nit: usually we would put this on one line for single imports:

`import {INotebookTracker} from @..`